src/components/sides/ManageLibrary.vue

```diff
@@ -233,10 +195,11 @@ export default {
     },
 
     productionUnsharedEntities() {
-      return this.unsharedAssets.filter(
-        entity => entity.project_id === this.productionId
-      )
+      return this.unsharedAssets
+        .filter(entity => entity.project_id === this.productionId)
+        .filter(entity => entity.entity_type_id === this.entityTypeId)
```

Comment:
Filters can be grouped together

```suggestion
      return this.unsharedAssets.filter(
        entity =>
          entity.project_id === this.productionId &&
          entity.entity_type_id === this.entityTypeId
      )
```